test/basic_test.cc

```diff
@@ -8,8 +8,7 @@
 
```

Comment:
i think you can use this in the other benchmarks too with the CalculatePi benchmark.